packages/astro/test/units/dev/collections-renderentry.test.js

```diff
@@ -0,0 +1,280 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+
+import { runInContainer } from '../../../dist/core/dev/index.js';
+import { createFsWithFallback, createRequestAndResponse } from '../test-utils.js';
+import mdx from '../../../../integrations/mdx/dist/index.js';
+
+const root = new URL('../../fixtures/content/', import.meta.url);
+
+describe.skip('Content Collections - render()', () => {
```

Comment:
can you make it so that these are only skipped on windows if that's where the problem is? I can take a look next week probably, but for now since they were passing on *nix it would be good to make them continue to be run there.

you can search the tests folder for `isWindows`, I think we have a couple that are skipped only on windows.